JavaScript/home.js

```diff
@@ -0,0 +1,6 @@
+const sandwich_menu = document.getElementsByClassName("sandwich_menu")[0]
+const menu = document.getElementsByClassName("menu")[0]

```

Comment:
I would try to avoid having my selectors rely on the position of the elements. 
This code will break if I add the "menu" class on another element somewhere above the current one